### PR TITLE
Fix: Updated various commands to utilize machinery

### PR DESCRIPTION
**Pull request by CGoodwin90**
# Checklist
- [ ] Affected Issues have been mentioned in the Closing issues section
- [ ] Documentation has been written/updated.
- [ ] Changelog entry has been written

Various commands are currently returning `Error: graphql: Syntax Error: Unexpected <EOF>` due to using a legacy lagoon client. Resolved utilizing `machinery` client & updated package to `v0.0.12`

- `listProjectByMetadata`
- `updateProjectMetadata`
- `deleteProjectMetadataByKey`
- `addDeployTargetConfigCmd`
- `getTaskByID`
- `runActiveStandbySwitch`

### CODE REVIEW

**Comment by shreddedbacon** [comment]
You should also remove the functions from the CLI internal packages if they're not used by anything else for anything that was replaced with machinery

File: cmd/deploytargetconfig.go [hunk 1]
```diff
@@ -48,31 +51,31 @@ var addDeployTargetConfigCmd = &cobra.Command{
...
 		if branches != "" {
 			addDeployTargetConfig.Branches = branches
 		}
-		if branches != "" {
+		if pullrequests != "" {
 			addDeployTargetConfig.Pullrequests = pullrequests
 		}
 		if deploytarget != 0 {
 			addDeployTargetConfig.DeployTarget = deploytarget
 		}
```
Comment (shreddedbacon) [inline cmd/deploytargetconfig.go:76]: this needs an `else` statement that returns an error saying that a deploytarget id must be provided

Comment (CGoodwin90) [reply cmd/deploytargetconfig.go:82]: Updated to check for the presence of deploytarget id & project

**Review by shreddedbacon — APPROVED** [review]
LGTM

**Pull request merged**